### PR TITLE
Test space creation and assigning user roles in CRD tests

### PR DESCRIPTION
**Pull request by matt-royal**
## Is there a related GitHub Issue?
 #1568 

## What is this change about?
 See issue

## Does this PR introduce a breaking change?
 The CRD tests now require 2 environment variables to run, so existing CI tasks and PR checks will likely fail until they are updated

## Acceptance Steps

- Run scripts/deploy-on-kind.sh kind
- Run tests/crds/run-crds-tests-kind.sh and confirm it passes

## Tag your pair, your PM, and/or team
 @tcdowney @clintyoshimura

### CODE REVIEW

File: tests/crds/crds_suite_test.go [hunk 1]
```diff
@@ -1,48 +1,84 @@
...
+	if skipSSL {
+		apiArguments = append(apiArguments, "--skip-ssl-validation")
+	}
+
+	Eventually(cf.Cf(apiArguments...)).Should(Exit(0))
+
+	// Stdin contains username followed by 2 return carriages. First one
+	// enters the username and second one skips the org selection prompt that
+	// is presented if there is more than one org
+	loginSession := cf.CfWithStdin(bytes.NewBufferString(user+"\n\n"), "login")
```
Comment (tcdowney) [inline tests/crds/crds_suite_test.go:82]: Out of curiosity, why do this instead of `cf auth user`?

Comment (matt-royal) [reply tests/crds/crds_suite_test.go:82]: I wasn't sure if `cf auth` had made it into a released version of the CF cli yet, so I just copied what the smoke tests do. If it's safe to use, then I'll switch it.

Comment (matt-royal) [reply tests/crds/crds_suite_test.go:82]: It looks like it's safe, so I'll make the change.

**Review by tcdowney — APPROVED** [review]
LGTM 👍

**Comment by matt-royal** [comment]
Close/reopen to re-trigger E2E tests

**Pull request merged**